examples/nimble_gatt/main.c

```diff
@@ -23,39 +23,26 @@
 #include <stdio.h>
 #include <string.h>
 
+#include "net/bluetil/ad.h"
+
 #include "host/ble_hs.h"
 #include "host/util/util.h"
 #include "host/ble_gatt.h"
 #include "services/gap/ble_svc_gap.h"
 #include "services/gatt/ble_svc_gatt.h"
 
-static const char device_name[] = "NimBLE on RIOT";
+static const char *device_name = "NimBLE on RIOT";
```

Comment:
While I like this version better, it isn't clear to me, why this change was made.